### ISSUE TITLE
no-setup-in-describe complains about arrow functions timeout

### ISSUE THREAD

**Issue by perrin4869** [lo1tuma/eslint-plugin-mocha#211]
The following code will error, even with `mocha/no-mocha-arrows` turned off:

```javascript
describe('my suite', () => {
  it('should pass within 2 seconds', () => { // mocha/no-setup-in-describe: Unexpected member expression in describe block. Member expressions may call functions via getters.
    // ...
  }).timeout(2000);
});
```

Maybe the solution would be to add an option to allow this use case?

**Comment by lo1tuma** [comment]
I would say this is a bug. It is similar to #208.
